examples/constants/constants.go

```diff
@@ -0,0 +1,41 @@
+
+//Event Signatures
+var TransferSignature = "0xddf252ad1be2c89b69c2b068fc378daa952ba7f163c4a11628f55a4df523b3ef"
```

Comment:
Future thought, we could easily derive these from a string of the method name and params.

Reply:
Good point. For anonymous events I don't think `topics[0]` is logged at all, how do you define a custom signature for them?